app/(blog)/components/BlogPost.tsx

```diff
@@ -1,4 +1,4 @@
-import { Card } from './card'
+import { Card } from '../../components/card'
```

Comment:
```suggestion
import { Card } from '../../components/card'
```

절대경로를 추가했으니 이걸 사용해보자.
근데 자동 import가 이걸로 안되는 이유는 더 찾아보자..-> 블로그 내용이 되지 않을까?